src/openforms/formio/rendering/conf.py

```diff
@@ -8,6 +8,9 @@
     RenderModes.pdf: RenderConfiguration(
         RenderConfigurationOptions.show_in_pdf, default=True
     ),
+    RenderModes.summary: RenderConfiguration(
+        RenderConfigurationOptions.show_in_summary, default=True
+    ),
```

Comment:
Is this something we want? `RenderConfigurationOptions.show_in_summary` was defined but `RenderModes.summary` is new.

Reply:
as discussed in the office - not at this time. The idea is that the summary should display everything the end-user has seen while filling out the form. This is not something that can be made conditional with logic (and is madness as a form builder to keep track of)

Reply:
Cool, I'll disable it for now and update the PR.